packages/mesh/src/Mesh.ts

```diff
@@ -477,20 +505,21 @@ export class Mesh extends Container
             this.geometry.dispose();
         }
 
-        this.geometry = null;
+        (this as any).geometry = null;
```

Comment:
@eXponenta changed geometry to private field + readonly prop, but for `destroy` both ways are fine and I want to underline that by approving this PR :)